hl13/code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/gun/ballistic/shotgun/hl13/spas12
```

Comment:
you did it right here by putting it inside the shotgun.dm file

Reply:
yep